addon_service/common/waterbutler_compat.py

```diff
@@ -24,15 +24,19 @@ class JSONAPIMeta:
 
     def _credentials_for_waterbutler(self, configured_storage_addon):
         _creds_data = configured_storage_addon.credentials
+        config = configured_storage_addon.config
+        host_url = config.external_api_url
+
         match type(_creds_data):
             case credentials.AccessTokenCredentials:
-                return {"token": _creds_data.access_token}
+                return {"token": _creds_data.access_token, "host": host_url}
```

Comment:
this quirk is only characteristic to owncloud, no need to add this for token auth